bios/samshiah-bio.md

```diff
@@ -0,0 +1,6 @@
+# About samshiah
+
+- Lives in San Francisco, CA
+- Avid Bay Area sports fan (Warriors, 49ers, Giants)
+- Fluent in two languages (Mandarin and English)
+- Crazy about dogs
```

Comment:
@molly-holly I love them all! 

@bobamoscms I just dogsat a mini golden this past weekend, super cute!